packages/base/src/3dview/mainview.tsx

```diff
@@ -362,29 +362,33 @@ export class MainView extends React.Component<IProps, IStates> {
 
       this._transformControls.enabled = false;
       this._transformControls.visible = false;
+      this._createViewHelper();
+    }
+  };
 
-      // ViewHelper setup
-      this._viewHelper = new ViewHelper(
-        this._camera,
-        this._renderer.domElement
-      );
-      this._viewHelper.center = this._controls.target;
+  private _createViewHelper() {
+    const viewHelperDiv = document.getElementById('viewHelper');
+    if (viewHelperDiv) {
+      viewHelperDiv.remove();
+    }
```

Comment:
This will remove the view helper of all opening documents. This is the expected behavior?

Reply:
Yeah, we can probably find a solution here that does not require using an `id` whatsoever, just `this.something.remove()`, something being the view helper container kept as a class property?

Reply:
ThankYou both, I agree this was not the expected behaviour even if it was not noticeable while testing for multiple documents open.

I also realised that it was not necessary to remove viewHelperDiv and just updating it with `updateCamera` works as well.


Done in [`39f6648`](https://github.com/jupytercad/JupyterCAD/pull/469/commits/39f664819e084b58636c54d0a203ad4570cf3820)